xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -56,7 +56,7 @@ To be able to format the output with indentation and white space OR to skip vali
       <Docs>
         <param name="bufferWriter">The destination for writing JSON text.</param>
         <param name="options">Defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra whitespace) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
-        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <see langword="System.Buffers.IBufferWriter&lt;System.Byte&gt;" />.</summary>
```

Comment:
I don't know why langword was chosen for this.

Reply:
I think @rpetrusha [did this as a workaround](https://github.com/dotnet/dotnet-api-docs/pull/2576) to correctly render a link with a specific T. It won't show you a link, it will show up like `true` or `false`, but at least it will render fine.

Your other options are what you did (use `1) or do what I did with Intrinsics, add a link but explain the expected T separately (explained in your other PR).